include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -309,7 +311,7 @@ struct layout_stride {
         size_t prev_stride_times_prev_extent =
           this->extents().extent(*next_idx_iter) * this->stride(*next_idx_iter);
         // "remove" the index
-        constexpr size_t removed_index_sentinel = -1;
+        constexpr size_t removed_index_sentinel = size_t(-1);
```

Comment:
What was wrong before?
```suggestion
        constexpr auto removed_index_sentinel = static_cast<size_t>(-1);
```

Reply:
some warning